### PR TITLE
add mongo to system packages for tools-jenkins

### PR DESCRIPTION
**Pull request by hchen15**
Configuration Pull Request
---

Make sure that the following steps are done before merging:

  - [ ] A DevOps team member has approved the PR.
  - [ ] Are you adding any new default values that need to be overridden when this change goes live? If so:
    - [ ] Update the appropriate internal repo (be sure to update for all our environments)
    - [ ] If you are updating a secure value rather than an internal one, file a DEVOPS ticket with details.
    - [ ] Add an entry to the CHANGELOG.
  - [ ] Have you performed the proper testing specified on the [Ops Ansible Testing Checklist](https://openedx.atlassian.net/wiki/display/EdxOps/Ops+Ansible+Testing+Checklist)?

### CODE REVIEW

File: playbooks/roles/tools_jenkins/defaults/main.yml [hunk 1]
```diff
@@ -78,3 +78,12 @@ jenkins_tools_debian_pkgs:
     - psmisc
     - mysql-client-core-5.6
     - ruby-sass
+
+MONGODB_APT_KEY: "7F0CEB10"
```
Comment (feanil) [inline playbooks/roles/tools_jenkins/defaults/main.yml]: Why is this in the tools-jenkins role? can we make a mongo-client role that encapsulates this that tools_jenkins depends on?

File: playbooks/roles/mongo_client/defaults/main.yml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+---
+MONGODB_APT_KEY: "7F0CEB10"
+MONGODB_APT_KEYSERVER: "keyserver.ubuntu.com"
+MONGODB_REPO: "deb http://repo.mongodb.org/apt/ubuntu xenial/mongodb-org/3.2 multiverse"
+mongo_version: 3.2.12
```
Comment (feanil) [inline playbooks/roles/mongo_client/defaults/main.yml]: Change this to `mongo_client_version`

File: playbooks/roles/mongo_client/defaults/main.yml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+---
+MONGODB_APT_KEY: "7F0CEB10"
+MONGODB_APT_KEYSERVER: "keyserver.ubuntu.com"
+MONGODB_REPO: "deb http://repo.mongodb.org/apt/ubuntu xenial/mongodb-org/3.2 multiverse"
```
Comment (feanil) [inline playbooks/roles/mongo_client/defaults/main.yml]: Change this to `MONGO_CLIENT_MONGODB_REPO`

File: playbooks/roles/mongo_client/defaults/main.yml [hunk 4]
```diff
@@ -0,0 +1,9 @@
+---
+MONGODB_APT_KEY: "7F0CEB10"
+MONGODB_APT_KEYSERVER: "keyserver.ubuntu.com"
```
Comment (feanil) [inline playbooks/roles/mongo_client/defaults/main.yml]: Change this to `MONGO_CLIENT_MONGODB_APT_KEYSERVER`

File: playbooks/roles/mongo_client/defaults/main.yml [hunk 5]
```diff
@@ -0,0 +1,9 @@
+---
+MONGODB_APT_KEY: "7F0CEB10"
```
Comment (feanil) [inline playbooks/roles/mongo_client/defaults/main.yml]: Change this to `MONGO_CLIENT_MONGODB_APT_KEY`

**Review by feanil — CHANGES_REQUESTED** [review]

**Review by fredsmith — APPROVED** [review]

**Review by feanil — APPROVED** [review]

**Pull request merged**